router/image/conf.d/nginx.conf.toml

```diff
@@ -12,6 +12,7 @@ keys = [
   "/deis/builder",
   "/deis/store/gateway",
   "/deis/certs",
+  "/registry/services/",
```

Comment:
Why are we using this path here? Are these keys all related to the `deis-registry` component? If not, maybe we should consider changing this.